compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -901,7 +907,8 @@ lDivRemGenericEvaluator64(TR::Node * node, TR::CodeGenerator * cg, bool isDivisi
    dependencies->addPostCondition(remRegister, TR::RealRegister::LegalEvenOfPair);
    dependencies->addPostCondition(quoRegister, TR::RealRegister::LegalOddOfPair);
 
-   if (!doConditionalRemainder &&
+   // TODO(#3686): Remove this Java-ism from OMR and push it into OpenJ9
```

Comment:
Sorry to be a pest.  The issue number is actually #3685.  This PR is #3686.  Ditto for the other comment below.

Reply:
Fixed in 36995ad